{{cookiecutter.module_name}}/terraform/main.tf

```diff
@@ -0,0 +1,20 @@
+provider "github" {}
```

Comment:
We shouldn't need these terraform configs. We can use `terraform init -from-module ...` to instantiate the code from the remote source.

* https://www.terraform.io/docs/commands/init.html#copy-a-source-module

We can also simply `git clone` the module, then `terraform init`

Reply:
sitting on this one b/c I'm planning on moving the git logic to [gitpython](https://github.com/gitpython-developers/GitPython). Same goes for the comment regarding the `org_repo.sh` file.